Resources/views/economy_entry/_economy_entry_service_table.html.twig

```diff
@@ -14,7 +14,7 @@
                 <tr>
                     <td>
                         {% if is_granted('edit', process) %}
-                            <a href="{{ path('economy_entry_edit', {'id': economyEntry.id, 'process': process}) }}">{{ economyEntry.service }}</a>
+                            <a href="{{ path('economy_entry_edit', {'id': economyEntry.id, 'process': process}) }}">{{ economyEntry.service ? economyEntry.service : 'economy.table.service_not_chosen'|trans }}</a>
```

Comment:
Can be written as `economyEntry.service ?: 'economy.table.service_not_chosen'|trans` (cf. https://twig.symfony.com/doc/3.x/templates.html#other-operators)